plugins/circleci/src/util/time.ts

```diff
@@ -31,5 +31,7 @@ export function durationHumanized(
   startTime: DateTimeObject,
   endTime: DateTimeObject,
 ) {
-  return dayjs.duration(dayjs(startTime).diff(dayjs(endTime))).humanize();
+  return dayjs
+    .duration(dayjs(startTime).diff(dayjs(endTime || new Date())))
```

Comment:
Is the plan to refactor this to use `luxon` instead of `dayjs`?

Reply:
yes, it's! 👍 
in this first PR, I'm just removing the `momentjs` lib, then we only have one library to care about when actually replacing 

(adding the `new Date()` in here is to cover the use case when a job in CircleCI is not done, thus endTime is `null`)